site/.vitepress/theme/Layout.vue

```diff
@@ -1,84 +1,125 @@
-      </a>
-      <div class="flex-auto"></div>
-      <a
-        class="py-1 px-3 border border-neon-yellow rounded-3xl text-2xl text-neon-yellow"
-        href="https://testnet.beamerbridge.com"
-        target="_blank"
-        >App</a
-      >
-      <a class="text-2xl text-sea-green" href="https://docs.beamerbridge.com" target="_blank"
-        >Documentation</a
-      >
+      <div class="absolute top-1/4 left-1/3 -z-10 h-1/2 w-full bg-radial-teal"></div>
+
+      <div class="container mx-auto flex flex-row items-center">
+        <a href="/">
+          <img class="w-64" src="/logo.png" />
+        </a>
+        <div class="flex-auto"></div>
+        <div class="flex flex-row items-center gap-16">
+          <a
+            class="rounded-3xl border border-neon-yellow py-1 px-3 text-xl text-neon-yellow transition-colors hover:bg-neon-yellow hover:text-dark-teal"
+            href="https://testnet.beamerbridge.com"
+            target="_blank"
+            >dApp</a
+          >
+          <a
+            class="text-xl text-sea-green transition-all hover:text-neon-yellow"
+            href="https://docs.beamerbridge.com"
+            target="_blank"
+            >Docs</a
+          >
+          <a
+            class="text-xl text-sea-green transition-all hover:text-neon-yellow"
+            href="https://docs.beamerbridge.com/faq.html"
+            target="_blank"
+            >FAQs</a
+          >
+        </div>
+      </div>
     </header>
 
-    <header class="lg:hidden w-screen flex flex-col items-center fixed z-20 bg-black">
-      <div class="m-4 p-2">
-        <div class="absolute -z-10 left-0 h-1/4 w-full bg-radial-teal"></div>
-        <img class="h-11" src="/logo.png" />
+    <header class="fixed z-30 flex h-[140px] w-screen flex-col items-center bg-black md:hidden">
+      <div class="my-auto">
+        <div class="absolute left-0 -z-10 h-1/4 w-full bg-radial-teal"></div>
+        <a href="/">
+          <img class="h-11" src="/logo.png" />
+        </a>
       </div>
       <div class="flex w-full">
         <a
           href="https://testnet.beamerbridge.com"
           target="_blank"
-          class="w-2/4 flex justify-center py-2 text-3xl bg-neon-yellow text-dark-teal"
-          >App</a
+          class="flex w-2/4 justify-center bg-neon-yellow py-2 text-2xl text-dark-teal"
+          >dApp</a
         >
         <a
           href="https://docs.beamerbridge.com"
           target="_blank"
-          class="w-2/4 flex justify-center py-2 text-3xl bg-sea-green text-dark-teal"
+          class="flex w-2/4 justify-center bg-sea-green py-2 text-2xl text-dark-teal"
           >Docs</a
         >
+        <a
+          href="https://docs.beamerbridge.com/faq.html"
+          target="_blank"
+          class="flex w-2/4 justify-center bg-[white] py-2 text-2xl text-dark-teal"
+          >FAQs</a
+        >
       </div>
     </header>
 
-    <div class="pt-20 lg:pt-24">
+    <div class="pt-[140px] md:pt-[105px]">
```

Comment:
That's true, i kinda overlooked the benefits of using `rem` instead of `px` because i was too focused on making a PIXEL perfect implementation of the copy :smile: 

Will switch to `rem`. Thanks for the heads up.